go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/fsouza/fake-gcs-server v1.38.4
 	github.com/googleapis/gax-go/v2 v2.11.0
 	github.com/jacobsa/daemonize v0.0.0-20160101105449-e460293e890f
-	github.com/jacobsa/fuse v0.0.0-20230509090321-7263f3a2b474
```

Comment:
did we fix anything in fuse library or just picked up with the latest fuse version?

Reply:
https://github.com/jacobsa/fuse/pull/149